### PR TITLE
add test comment

### PR DESCRIPTION
**Pull request by gitboej**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,3 +55,9 @@ async function fetchCharacters() {
     console.error(error);
   }
 }
+
+/**
+ *
+ * This is just a test to show the workflow for PR comments and approvals.
+ *
+ */
```
Comment (gitboej) [inline index.js:63]: nice comment

File: index.js [hunk 2]
```diff
@@ -14,12 +14,10 @@ let maxPage = 1;
 let page = 1;
 let searchQuery = "";
 
-
 const prevButton = createButton("previous", () => {
   if (page <= 1) return;
   page--;
   fetchCharacters();
-
 });
```
Comment (benkutdev) [inline index.js:21]: Love that fetch!

**Review by gitboej — COMMENTED** [review]
pls have a look at my suggestion

**Review by benkutdev — APPROVED** [review]
fine!

**Pull request merged**